exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricTranslator.java

```diff
@@ -54,6 +59,12 @@
   static final Set<MetricDataType> DOUBLE_TYPES = ImmutableSet.of(DOUBLE_GAUGE, DOUBLE_SUM);
   private static final int MIN_TIMESTAMP_INTERVAL_NANOS = 1000000;
 
+  private static final Map<String, AttributeKey<String>> gceMap =
```

Comment:
Might be useful to leave a comment with a link to https://cloud.google.com/monitoring/api/resources#tag_gce_instance as the source of truth.